### ISSUE TITLE
Fix FBSDKLoginKit dependency for AWSFacebookSignIn

### ISSUE THREAD

**Issue by rohandubal** [aws-amplify/aws-sdk-ios#1397]
For release 2.9.4 the pod publish failed due to new FBSDK release 4.43+. Need to root cause and fix.

**Comment by BillBunting** [comment]
Current versions continue to move forward, but AWS `AWSFacebookSignIn` does not.  I'd like to be able to use current features in FBSDKShareKit but am unable because of `AWSFacebookSignIn`

```
- FBSDKCoreKit 4.42.0 -> 4.42.0 (latest version 5.7.0)
- FBSDKLoginKit 4.42.0 -> 4.42.0 (latest version 5.7.0)
- FBSDKShareKit 4.42.0 -> 4.42.0 (latest version 5.7.0)
```

```
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In snapshot (Podfile.lock):
    FBSDKCoreKit (<= 4.42, = 4.42.0)

  In Podfile:
    AWSFacebookSignIn (~> 2.11.1) was resolved to 2.11.1, which depends on
      FBSDKCoreKit (<= 4.42)

    FBSDKShareKit (~> 5.6.0) was resolved to 5.6.0, which depends on
      FBSDKShareKit/Share (= 5.6.0) was resolved to 5.6.0, which depends on
        FBSDKCoreKit (~> 5.0)
```

**Comment by royjit** [comment]
Updated the FB SDK to 5.8. This has been released in the version [2.12.1](https://github.com/aws-amplify/aws-sdk-ios/releases/tag/2.12.1)
